parsec/mca/device/device.c

```diff
@@ -219,14 +224,15 @@ int parsec_select_best_device( parsec_task_t* this_task ) {
 
         /* Consider how adding the current task would change load balancing
          * between devices */
+        if(!parsec_device_load_balance_allow_cpu)
+            valid_types &= ~PARSEC_DEV_CPU; /* automatic CPU / GPU load balancing disabled, remove the CPU type */
+        valid_types &= ~PARSEC_DEV_RECURSIVE; /* Recursive device time estimates are computed on the associated CPU device */
```

Comment:
`Recursive` will be disabled now even if `parsec_device_load_balance_allow_cpu` is true, right?

Reply:
I see. This is only for time estimation.